### PR TITLE
docs: Clarify weighted fair-share scheduling policy

### PR DESCRIPTION
**Pull request by tara-det-ai**
The description of fair-share scheduling is too hard to read. Rewrite it so that users can better understand the meaning and how to modify its behavior.

## Description

The description of fair-share scheduling is difficult to comprehend.

## Checklist

- [ ] Changes have been manually QA'd
- [ ] Meets style guidelines

### CODE REVIEW

File: docs/architecture/introduction.rst [hunk 1]
```diff
@@ -491,17 +491,22 @@ scheduling behavior of an individual task is influenced by several task configur
...
-*demand* (number of desired concurrent tasks) of each experiment. For instance, in an eight-GPU
-cluster running two experiments with demands of ten and thirty, the scheduler assigns two slots and
-six slots respectively. As new experiments become active or the resource demand of an active
-experiment changes, the scheduler will adjust how slots are allocated to experiments as appropriate.
-
-The behavior of the fair-share scheduler can be modified by changing the *weight* of a workload. A
+fair-share scheduling policy. This policy aims for fair distribution of resources, taking into
+account each experiment's request. More specifically, slots are divided among the active experiments
+according to the demand of each experiment, where *demand* is the number of desired concurrent
+tasks.
```
[the remark this answers is not in the document]
Comment (tara-det-ai) [reply docs/architecture/introduction.rst:501]: thanks @ioga ! i think it's worth explaining,
@rb-determined-ai what do you think?

[the remark this answers is not in the document]
Comment (tara-det-ai) [reply docs/architecture/introduction.rst:501]: does this capture what you described? just trying to understand it.
fair-share scheduler algorithm explanation

[the remark this answers is not in the document]
Comment (tara-det-ai) [reply docs/architecture/introduction.rst:501]: discussed via slack. we'll put these details elsewhere (keeping them out of this high level doc)

**Review by ighodgao — COMMENTED** [review]
I am still confused - why did the two/six slot split change to one/seven? And why does the demand=30 experiment get 7 slots? Is it not assigned proportionally to demand (e.g. 1/3 split)?

**Comment by ioga** [comment]
hmm, did I get confused on this as well? basically it should be proportional for single-slot experiments but <a complicated mess> for dtrain.

**Pull request merged**